### PR TITLE
Update the detection method for docker overlay2 issue

### PR DESCRIPTION
**Pull request by wangzhen127**
This PR updates the detection method for docker overlay2 issue. We no longer count how many times the log message is seen.

### CODE REVIEW

File: config/docker-monitor.json [hunk 1]
```diff
@@ -9,11 +9,24 @@
 	"source": "docker-monitor",
 	"metricsReporting": true,
 	"conditions": [],
```
Comment (Random-Liu) [inline config/docker-monitor.json]: 2 conditions?

File: config/docker-monitor.json [hunk 2]
```diff
@@ -9,11 +9,24 @@
...
+	"conditions": [
```
Comment (Random-Liu) [inline config/docker-monitor.json]: indent?

**Comment by yguo0905** [comment]
Remove the one from docker-monitor-counter.json?

File: config/docker-monitor.json (hunk 1 above)
Comment (wangzhen127) [reply config/docker-monitor.json]: fixed

File: config/docker-monitor.json (hunk 2 above)
Comment (wangzhen127) [reply config/docker-monitor.json]: done

**Comment by wangzhen127** [comment]
> Remove the one from docker-monitor-counter.json?

I cannot  remove it now. The presubmit and CI jobs are using k8s master head to test NPD e2e tests, which needs docker-monitor-counter.json. We can remove this config file after updating the k8s master.

**Pull request merged**